hikari/impl/rest.py

```diff
@@ -1204,26 +1204,31 @@ async def fetch_message(
         assert isinstance(response, dict)
         return self._entity_factory.deserialize_message(response)
 
-    async def _create_message(
+    def _build_message(
```

Comment:
```suggestion
    def _build_message_payload(
```

Just for a little more clarity